src/openrct2/localisation/Formatting.cpp

```diff
@@ -576,6 +576,23 @@ namespace OpenRCT2
                     }
                 }
                 break;
+            case FormatToken::Height:
+                if constexpr (std::is_integral<T>())
+                {
+                    auto metres = HeightUnitsToMetres(arg);
```

Comment:
If `gConfigGeneral.ShowHeightAsUnits` is enabled, what does the ride stat window show? Does it show the drop height as metres/feet or units? If it is the latter, this should be reflected here.

Reply:
The ride stat window doesn't show height measurements in units, only metres/feet.